pkgs/development/compilers/nim/default.nix

```diff
@@ -48,35 +47,53 @@ stdenv.mkDerivation rec {
                  -d:useGnuReadline \
                  ${lib.optionals (stdenv.isDarwin || stdenv.isLinux) "-d:nativeStacktrace"}
     ./koch tools -d:release
-  '';
 
-  installPhase = ''
-    install -Dt $out/bin bin/* koch
-    ./koch install $out
-    mv $out/nim/bin/* $out/bin/ && rmdir $out/nim/bin
-    mv $out/nim/*     $out/     && rmdir $out/nim
-    wrapProgram $out/bin/nim \
-      --suffix PATH : ${lib.makeBinPath [ stdenv.cc ]}
```

Comment:
This wrapper is still needed, I think.

Reply:
Does "nim compile" work for you in pure environment?

Reply:
No, it doesnt, you are right. I fixed it.